back/src/index.ts

```diff
@@ -1,5 +1,17 @@
 import 'regenerator-runtime/runtime';
+import * as fs from 'fs';
+import * as path from 'path';
 import { config } from 'dotenv';
+import colors from 'colors';
+try {
+  fs.readFileSync(path.join(__dirname, '..', '.env'), 'utf-8');
```

Comment:
Why do you need this code?

Reply:
Yes, it lunch a warning, but is not necessary , the documentation is very clear